### PR TITLE
Update SITE_CONTRIBUTION.md

### PR DESCRIPTION
**Pull request by gary-tai**
Update content for Hugo.

### CODE REVIEW

File: SITE_CONTRIBUTION.md [hunk 1]
```diff
@@ -1,166 +1,130 @@
...
+[3]: {{< param github_url >}}/issues
+[4]: https://httpbin.org/
+[5]: https://github.com/projectcontour/community/wiki/Office-Hours
+[6]: {{< param slack_url >}}
+[7]: https://github.com/bitnami/charts/tree/master/bitnami/contour
+[8]: https://www.youtube.com/watch?v=xUJbTnN3Dmw
+```
 
-Note that the version of [Jekyll][6] used by the site is pinned in the [Gemfile](./site/Gemfile.lock). The `site-devel` target uses the matching versioned Docker image to ensure compatibility.
+## Using URL parameters
```
Comment (skriss) [inline SITE_CONTRIBUTION.md:65]: I know the section just above shows examples of how to use these params, but I would include an example of how to use them in this section ("Using URL parameters") as well.

File: SITE_CONTRIBUTION.md [hunk 2]
```diff
@@ -1,166 +1,130 @@
...
+## Testing your changes
+To test your website changes, run the following command from the `site` directory:
+```
+hugo server
+```
 
-Install [Jekyll][6] and plug-ins in one fell
-swoop. This mirrors the plug-ins used by GitHub Pages on your local
-machine including Jekyll, Sass, etc:
+Go to `http:\\localhost:1313` to view the website and changes in your browser. When ready, commit and push your changes to GitHub and create a Pull Request (PR). You can let the team know in the [Contour Slack channel][7] that you have created a PR.
```
Comment (skriss) [inline SITE_CONTRIBUTION.md:110]:
```suggestion
Go to `http://localhost:1313` to view the website and changes in your browser. When ready, commit and push your changes to GitHub and create a Pull Request (PR). You can let the team know in the [Contour Slack channel][7] that you have created a PR.
```

**Review by skriss — CHANGES_REQUESTED** [review]
Thanks for this @gary-tai, this looks great! Just a couple small comments.

File: SITE_CONTRIBUTION.md (hunk 1 above)
Comment (gary-tai) [reply SITE_CONTRIBUTION.md:65]: added a few sample reference links.

File: SITE_CONTRIBUTION.md (hunk 2 above)
Comment (gary-tai) [reply SITE_CONTRIBUTION.md]: fixed to fwd slashes

**Comment by gary-tai** [comment]
updated with new commit.

**Comment by gary-tai** [comment]
Created changelogs/unreleased/4620-gary-tai-docs.md

**Comment by skriss** [comment]
Changes look good, the changelog file still needs to be added though, thanks!

**Comment by gary-tai** [comment]
Changelog PR should be there now...
https://github.com/projectcontour/contour/pull/4625

**Comment by youngnick** [comment]
Ah, it's intended that you add the CHANGELOG file to *this* PR once you have the PR number, that way everything is in one place.

**Review by skriss — APPROVED** [review]

**Pull request merged**